pyproject.toml

```diff
@@ -16,18 +16,20 @@ packages = [{include = "sycamore"}]
 python = ">=3.9,<3.12"
 
 [tool.poetry.group.sycamore_library.dependencies]
-opensearch-py = "^2.2.0"
+opensearch-py = "^2.3.1"
 pandas = "2.1.1"
 pdf2image = "^1.16.3"
 ray = { extras = ["default"], version = "^2.7.0" }
 sentence-transformers = "2.2.2"
 pyarrow = "13.0.0"
-unstructured = "0.10.19"
+unstructured = "0.10.20"
+openai = "0.28.1"
```

Comment:
Thanks for doing this. I spent 2 minutes looking at the newest version and it seems that they changed a fair amount. I will add an issue to upgrade properly, but in the meantime this is good.

Reply:
We should upgrade OpenAI also, maybe at the same time.  And any other similar packages.